src/strategies/hashes-voting/index.ts

```diff
@@ -0,0 +1,55 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'Arr00';
```

Comment:
This should be your username
```suggestion
export const author = 'brennanfife';
```